### ISSUE TITLE
Add 2i and MapReduce top query reporting to riak-admin

### ISSUE THREAD

**Issue by michellep-basho** [basho/riak#432]
Add the ability to report out on top MapReduce and 2i queries to riak-admin with various sorting parameters. For example: top indices hit, longest running queries, most frequently used queries.  Request from Evan and Richard S of Basho.

**Comment by jaredmorrow** [comment]
@michellep Is this still an active request?  Also, should this be tracked in our PM or platform_task repos?

**Comment by michellep-basho** [comment]
Yes, this has come up again recently with 2 other customers and 1 open source user. It doesn't necessarily have to be through riak-admin, but more visibility is needed. thanks

**Comment by binarytemple** [comment]
+1 It would be very useful to identify specific MR jobs. I see instances where one person is running a bunch of MR without informing their teammates.

**Comment by michellep-basho** [comment]
Same goes for the list key operations.
